packages/webpack-cli/bin/cli.js

```diff
@@ -4,13 +4,21 @@
 require('v8-compile-cache');
 const importLocal = require('import-local');
 const runCLI = require('../lib/bootstrap');
+const { yellow } = require('colorette');
+const { packageExists, promptInstallation } = require('@webpack-cli/package-utils');
 
 // Prefer the local installation of webpack-cli
 if (importLocal(__filename)) {
     return;
 }
 process.title = 'webpack';
 
-const [, , ...rawArgs] = process.argv;
-
-runCLI(rawArgs);
+if (packageExists('webpack')) {
```

Comment:
For `@webpack-cli/package-utils` to detect whether `webpack` exists, `webpack` should be listed as a peer dependency of `@webpack-cli/package-utils`, or at least as an optional peer dependency via `peerDependenciesMeta` (#1816).